docs/index.rst

```diff
@@ -0,0 +1,8 @@
+.. toctree::
+   :hidden:
+   :maxdepth: 2
+
+   api
+   examples
+
+.. mdinclude:: ../README.md
```

Comment:
Can you update the Readme to include installation instructions.

Reply:
Done.